lib/quick_travel/adapter.rb

```diff
@@ -102,21 +102,15 @@ def self.check_id!(id)
     end
 
     def self.find_all!(request_path, opts = {})
-      response = if opts.key? :cache
-        QuickTravel::Cache.cache(opts[:cache], opts[:cache_options]) {
-          get_and_validate(request_path, opts.except(:cache, :cache_options))
-        }
-      else
-        get_and_validate(request_path, opts, return_response_object: true)
-      end
-      full_response = response.respond_to? :parsed_response
-      parsed_response = full_response ? response.parsed_response : response
+      response = QuickTravel::Cache.cache(opts[:cache], opts[:cache_options]) {
```

Comment:
It's probably more clear if we use the keys `:cache_key` `:cache_options` instead of `:cache` and `:cache_options`